src/subscription/subscription.service.ts

```diff
@@ -124,15 +124,15 @@ export class SubscriptionService {
         where: { id: profileId },
       });
 
-      if (profileId === api.profileId) {
-        throw new BadRequestException(
-          ZaLaResponse.BadRequest(
-            'Bad request',
-            "You can't subscribe to your own api",
-            '400',
-          ),
-        );
-      }
+      // if (profileId === api.profileId) {
+      //   throw new BadRequestException(
```

Comment:
kindly remove commented codes if not useful anymore

Reply:
Yeah, I forgot to uncomment it, did that to check something